src/App.jsx

```diff
@@ -13,15 +13,115 @@ function App() {
+                }`,
+            },
+        };
+        try {
+            const response = await fetch(apiURL, options);
+
+            if (!response.ok) {
+                const message = `Error has ocurred: ${response.status}`;
+                throw new Error(message);
+            }
+            const data = await response.json();
+            const todos = data.records.map((todo) => {
+                const newTodo = {
+                    id: todo.id,
+                    title: todo.fields.title,
+                };
+                return newTodo;
+            });
+            setTodoList(todos);
             setIsLoading(false);
-        });
+        } catch (error) {
+            console.log(error.message);
+            return null;
+        }
+    };
+
+    // Post a task to Airtable
+    const postTodo = async (todo) => {
+        try {
+            const airtableData = {
+                fields: {
+                    title: todo.title,
+                },
+            };
+
+            const response = await fetch(
+                `https://api.airtable.com/v0/${
+                    import.meta.env.VITE_AIRTABLE_BASE_ID
+                }/Default`,
+                {
+                    method: "POST",
+                    headers: {
+                        "Content-Type": "application/json",
+                        Authorization: `Bearer ${
+                            import.meta.env.VITE_AIRTABLE_API_TOKEN
+                        }`,
+                    },
+                    body: JSON.stringify(airtableData),
+                }
+            );
+
+            if (!response.ok) {
+                const message = `Error has ocurred: ${response.status}`;
+                throw new Error(message);
+            }
+
+            const dataResponse = await response.json();
+            console.log(dataResponse);
+            // return dataResponse;
+            return { id: dataResponse.id, title: dataResponse.fields.title };
+        } catch (error) {
+            console.log(error.message);
+            return null;
+        }
+    };
+
+    // Delete from Airtable
+    const postDeleteTodo = async (id) => {
+        try {
+            const response = await fetch(
+                `https://api.airtable.com/v0/${
+                    import.meta.env.VITE_AIRTABLE_BASE_ID
+                }/Default/${id}`,
+                {
+                    method: "DELETE",
+                    headers: {
+                        "Content-Type": "application/json",
+                        Authorization: `Bearer ${
+                            import.meta.env.VITE_AIRTABLE_API_TOKEN
+                        }`,
+                    },
+                }
+            );
+
+            if (!response.ok) {
+                const message = `Error has ocurred: ${response.status}`;
+                throw new Error(message);
+            }
+
+            const data = await response.json();
+            return data;
```

Comment:
When we call this delete function, we don't use a returned value but there is one here. Let's pick one way or the other but not both.